src/main/java/net/sf/jabref/exporter/SavePreferences.java

```diff
@@ -51,7 +51,7 @@ public static SavePreferences loadForExportFromPreferences(JabRefPreferences pre
     }
 
     public static SavePreferences loadForSaveFromPreferences(JabRefPreferences preferences) {
-        Boolean saveInOriginalOrder = true;
```

Comment:
Hm, do I understand you correctly @tobiasdiez: The preference `saveInOriginalOrder` overwrites meta data in the file? If so, then this is a problem, because the serialization can differ between machines. That harms version-controllability, our top aim regarding the serialization.

I think meta data that is explicitly written into the file should always be followed above user-specific preferences.